src/assets/sass/protocol/base/elements/_links.scss

```diff
@@ -30,17 +30,17 @@
 
     &:link,
     &:visited {
-        border-bottom: 2px solid transparent;
-        color: $color-black;
+        border-bottom: 2px solid;
+        color: $color-link;
         text-decoration: none;
     }
 
     &:hover,
     &:active,
     &:focus {
         @include transition(border-bottom-color 100ms ease-in-out);
-        border-bottom: 2px solid $color-black;
-        color: $color-black;
+        border-bottom: 2px solid;
+        color: $color-link-hover;
```

Comment:
Not a blocker: I'm guessing that the hover states for the light and dark themes differ is intentional, but would making them more consistent be worth doing? e.g. the color doesn't change, but we do remove the bottom-border?